### ISSUE TITLE
More smart switches on the same ESP32?

### ISSUE THREAD

**Issue by Haimana** [SignalK/SensESP#697]
Hello, 

Is it possible to have more smart switches (push buttons and relays) controlled by same ESP32?
For saving pins, I'm thinking not to use the RGB leds for each switch and to have just a regular led attached on each output pin which is also used for driving the relay.
Any advice is welcome

Thank you,
Seb

**Comment by JohnySeven** [comment]
Yes, check my repo here: https://github.com/JohnySeven/Boat-T-RelayS3 it's based on T-RelayS3 board.
